src/main/java/com/smashingmods/alchemistry/common/block/dissolver/DissolverBlockEntity.java

```diff
@@ -154,7 +154,7 @@ protected void onEnergyChanged() {
     public ProcessingSlotHandler initializeInputHandler() {
         return new ProcessingSlotHandler(1) {
             @Override
-            protected void onContentsChanged(int slot) {
+            protected void onContentsChanged(int pSlot) {
```

Comment:
These small parameter changes are done to match code style and maintain consistency, trivial